client/app.jsx

```diff
@@ -163,13 +168,14 @@ class App extends React.Component {
       this.setState({
         availableDates: stay,
         checkOut: selectedDay + ' ' + monthYear,
-        bookStage: 'invoice'
+        currentPrice: this.state.availableDates[0].fee,
+        bookStage: 'invoice',
+        invoice: true
       })
-
-      //TODO must write logic to clear calendar when user wants to reselect dates
-      //TODO pull up invoice for remaining availableDates after selection
     }
 
+    //TODO must write logic to clear calendar when user wants to reselect dates
```

Comment:
I completely forgot to add TODO's in my work. Thanks for the reminder. It makes things more organized for you and the rest of the team!